sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -55,7 +56,7 @@ object SQLConf {
   val WAREHOUSE_PATH = SQLConfigBuilder("spark.sql.warehouse.dir")
     .doc("The default location for managed databases and tables.")
     .stringConf
-    .createWithDefault("file:${system:user.dir}/spark-warehouse")
+    .createWithDefault("${system:user.dir}/spark-warehouse")
```

Comment:
Yes, we can use `File.toURI()` for `WAREHOUSE_PATH`, if we are sure that it is always a local path. However, I remember someone in this thread mentioned that the path might be an amazon `s3` path. Is this supposed to happen?